exercises/nth-prime/example.lua

```diff
@@ -0,0 +1,18 @@
+return function(n)
+  assert(n > 0, 'n must be positive')
+
+  local primes = {}
+
+  local function is_prime(n)
+    for _, p in ipairs(primes) do
+      if n % p == 0 then return false end
+      if p^2 > n then return true end
```

Comment:
Do you mean is there a way to check that the performance is on the right order of magnitude? If so you can use the `os` package to roughly measure time.